packages/story-editor/src/components/canvas/multiSelectionMoveable/useResize.js

```diff
@@ -53,13 +56,17 @@ function useResize({ onGroupEventEnd, targetList, setTransformStyle, frames }) {
       if (updates && updates.height) {
         newHeight = dataToEditorY(updates.height);
       }
-      target.style.width = `${newWidth}px`;
-      target.style.height = `${newHeight}px`;
+
+      // We add the border size back for the target display.
+      const frameWidth = newWidth + left + right;
+      const frameHeight = newHeight + top + bottom;
+      target.style.width = `${frameWidth}px`;
+      target.style.height = `${frameHeight}px`;
```

Comment:
Somewhere around here there's a bug when resizing a bordered element as part of multi-select. You can make it negative size!
![negative-size](https://user-images.githubusercontent.com/637548/181546340-fc452775-1eb9-4249-96dd-f7bec2f7e6f0.gif)

Reply:
This was a general bug with the multi-selection moveable -- it didn't consider the min width/height set for each element. Fixed now.